src/osgPlugins/fbx/WriterNodeVisitor.h

```diff
@@ -45,8 +45,8 @@ struct Triangle
 
 struct VertexIndex
 {
```

Comment:
I am not seeing any changes to this file in 3.4 so we should probably take the 3.6 version of the file.

Reply:
Reverted changes to this file (pending)